extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -191,66 +191,64 @@ private function fetchAll($useMasterDb = false) {
 				}
 				*/
 
-				if($row->type == BADGE_TYPE_INTRACKEDITPLUSCATEGORY)
-					$this->mInTrackEditPlusCategory[$row->id] = array('enabled' => $row->enabled, 'category' => $row->cat);
-				elseif($row->type == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM) {
-					$this->mNotInTrackCommunityPlatinum[$row->id] = array(
+				if( $row->type == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
+					$this->mInTrackEditPlusCategory[ $row->id ] = array( 'enabled' => $row->enabled, 'category' => $row->cat );
+				elseif( $row->type == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM ) {
+					$this->mNotInTrackCommunityPlatinum[ $row->id ] = array(
 						'enabled' => $row->enabled,
 						'is_sponsored' => $row->sponsored,
 						'badge_tracking_url' => $row->badge_tracking_url,
 						'hover_tracking_url' => $row->hover_tracking_url,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "hover_tracking_url" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)